### ISSUE TITLE
Gmail new/blank lines bug

### ISSUE THREAD

**Issue by PrestanceDesign** [akahuku/wasavi#134]
Hi,

Since the last version, all empty lines are removed from Gmail when saving. It seems like this issue #117.
I have rolled back with previous version of Wasavi and all is good now.

Best regards.

**Comment by yankcrime** [comment]
Yup, I'm seeing this as well.

**Comment by PrestanceDesign** [comment]
I close this issue because it's duplicated from #31.

**Comment by yankcrime** [comment]
I think a new issue for this was appropriate given that it appears to be a regression.  As it stands right now there's two issues but they're both closed so I'm not sure they'll receive any attention.

**Comment by PrestanceDesign** [comment]
You're right @yankcrime, I reopen this issue.

I have captured the HTML of my message both before and after editing/saving with wasavi. Maybe this can help to debug.
We can see a `br` tag inside a `div` has been removed. 

### Before :
``` html
<div id=":93" class="Am aO9 Al editable LW-avf" hidefocus="true" aria-label="Corps du message" g_editable="true" role="textbox" contenteditable="true" tabindex="1" style="direction: ltr; min-height: 85px;">This is the first line.<div><br></div><div>This is the second line.</div></div>
```

### After :
``` html
<div id=":93" class="Am aO9 Al editable LW-avf" hidefocus="true" aria-label="Corps du message" g_editable="true" role="textbox" contenteditable="true" tabindex="1" style="direction: ltr; min-height: 85px;"><div>This is the first line.</div>
<div>This is the second line.</div>
</div>
```

**Comment by RichardDooling** [comment]
Yes, if I put blank lines between paragraphs they disappear on :wq

Rick

**Comment by igavronski** [comment]
Same problem here. Is someone looking at this problem?

**Comment by PrestanceDesign** [comment]
I downgraded and blocked to the version 0.6.628 my Chrome extension until a fix will be released.

When I have a little bit of time, I would like to compare the versions to determine the cause of the problem. I guess it's the new feature **"content editable element to markdown"** that cause this issue.
https://github.com/akahuku/wasavi/wiki/Editing-%22content-edltable%22-element

**Comment by igavronski** [comment]
Sorry, but I don't know and I couldn't find how to downgrade an extension. Could you please provide (a link for) instructions?

**Comment by igavronski** [comment]
By the way, I followed the link to the wiki you provided, some trial and error, and got it working on Gmail by  setting `set writeas=textAndBreak` in the exrc.

**Comment by PrestanceDesign** [comment]
Are you on Chrome ? 
I yes, you can download the version 0.6.628 from [this link ](https://github.com/akahuku/wasavi/releases/download/v0.6.628/wasavi.crx) and follow method 2 from [here](https://blog.hunter.io/install-chrome-extension-manually/).

**Comment by PrestanceDesign** [comment]
Oh I just try and it works. I had missed these options !
I continue to testing tomorrow and if it works without problem, I'll close this issue.

**Comment by ardent246** [comment]
I was curious if there has been any progress on this? All blank lines are being deleted by wasavi in Gmail in version 0.7.737. Putting set writeas=textAndBreak does not make any difference for me. Thanks!
